ci_scripts/ci_post_clone.sh

```
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/zsh
```

Shellcheck

[error] ShellCheck only supports sh/bash/dash/ksh scripts. Sorry!

(SC1071)